src/kvirc/kernel/KviIrcSocket.cpp

```diff
@@ -1393,7 +1393,7 @@ void KviIrcSocket::doSSLHandshake(int)
 	}
 
 #else //COMPILE_SSL_SUPPORT
-	qDebug("Ops.. SSL handshake without SSL support! Aborting!");
+	qDebug("Ooops... SSL handshake without SSL support! Aborting!");
```

Comment:
Theres a few Ooops... already so I rather make them all the same whatever that is.